src/defaultsettings.cpp

```diff
@@ -71,6 +71,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("keymap_special1", "KEY_KEY_E");
 	settings->setDefault("keymap_chat", "KEY_KEY_T");
 	settings->setDefault("keymap_cmd", "/");
+	settings->setDefault("keymap_cmd_local", ".");
```

Comment:
@SmallJoker  Done.
@nerzhul other key mappings are in `minetest.conf.example` and `settingtypes.txt`.